packages/frontend-shared/src/gql-components/topnav/LoginModal.vue

```diff
@@ -42,7 +42,7 @@
               href="https://on.cypress.io/dashboard/profile"
               class="font-medium text-indigo-500"
             >
-              {{ viewer.fullName }}
+              {{ viewer.fullName || viewer.email }}
```

Comment:
nice 👍🏻  are users allowed to create a user without a full name though?

Reply:
Maybe not anymore, but I believe there are older accounts that could be created without associated names. Both fullName and email are optional within the CloudUser model, but from my understanding email should always be present.

Reply:
oh good to know 👍